packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -388,6 +388,10 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 			this.sortData();
 		}
 
+		this.paginationConfig = {
+			...this.paginationConfig,
+			totalItems: this.dataSource ? this.dataSource.filteredData.length : this.data.length
+		};
```

Comment:
Could you write some unit tests for this? Otherwise we have no means to know whether we solved the issue. And also to be sure that we will not break it again while developing another feature ;)

Reply:
I wrote a couple of tests and added them to the PR. 😄